tests/e2e/test_notebook.py

```diff
@@ -0,0 +1,27 @@
+from unittest import TestCase, SkipTest
+import subprocess
+import os
+import shutil
+
+class AutoCyberE2ETest(TestCase):
+    slow = 0
+
+    def test_e2e_autocyber_tutorial(self):
+        try:
+            subprocess.check_call(['ipython', '-V'])
+        except:
+            raise SkipTest('IPython is not installed')
+
+        nb_file = os.path.join(os.path.dirname(__file__), "AutoCyberTutorial.ipynb")
+        nb_folder = "examples/AutomotiveCybersecurity/notebooks"
+
+        shutil.copy2(nb_file, nb_folder)
```

Comment:
we can use `tempfile.TemporaryDirectory` if you want to create a directory to be clean by the end.